src/components/Header.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+
+const Header = () => {
+  const [darkMode, setDarkMode] = useState(false);
+
+  const displayDarkMode = () => {
+    setDarkMode(!darkMode);
+  };
+
+  useEffect(() => {
+    if (darkMode) document.querySelector('body').classList.add('dark-theme');
+    else document.querySelector('body').classList.remove('dark-theme');
+  }, [darkMode]);
+
+  return (
+    <>
+      <header className='header'>
+        <div className='container flex flex-jc-sb flex-ai-c'>
+          <div className='logo'>
+            <button
+              className='regular-button'
+              onClick={() => (window.location.href = 'index.html')}
+            >
```

Comment:
It would be better to extract the function outside of the jsx and just call it in here